### ISSUE TITLE
Master Levels force pistol start?

### ISSUE THREAD

**Issue by OpenRift412** [fabiangreffrath/crispy-doom#793]
Doom II's Master Levels were all designed/selected with pistol-starting in mind, so maybe when Crispy detects masterlevels.wad or one of the individual PWADs, it forces the -pistolstart parameter upon level load. Just a thought.

**Comment by fabiangreffrath** [comment]
Nope, sorry. In masterlevels.wad the maps are arranged into one consecutive megawad and there are speedruns available which run through all of them in order. If you want pistol starts, there is the corresponding command line parameter, but I am not going to second-guess that upon the players because "I know how these levels were meant to be played".

**Comment by NightFright2k19** [comment]
Use International Doom. You can force pistol starts after each level.

**Comment by Dark-Jaguar** [comment]
Personally, I always found masterlevels.wad to be a pretty awkward badly layed out megawad in the first place, since there's basically two partly complete level sets within it as well as a number of levels never designed with any sort of real difficulty curve in mind.  They're organized alphabetically by name, which also messes with the flow.  This isn't a criticism of Crispy Doom.  Your support for even this wad is very much appreciated, but my preferred way to play them is independent of each other through something like https://zdoom.org/wiki/Master_Levels_Menu_Interface#:~:text=The%20Master%20Levels%20Menu%20Interface,through%20an%20in%2Dgame%20menu.
Which unfortunately does not and cannot work with Crispy.  I may make a batch file some time to handle each level and point to the right "level start" for each WAD, but that leaves out level descriptions.  Eh, oh well it is what it is.

**Comment by NightFright2k19** [comment]
I agree, they should have tried at least to group levels by authors to have some sort of consistency regarding design. Then again, when trying to compile levels as an episode which most of them weren't meant for, you have to make compromises. 

At the end of the day, it's still best to load these maps individually and play them the way they were intended. Anyway, regardless how you go about it, most of these maps just suck and no rearrangement you can possibly do will make them any better.
